nucliadb_node/src/shards/shard_writer.rs

```diff
@@ -52,6 +53,11 @@ pub struct ShardWriter {
     write_lock: Mutex<()>,  // be able to lock writes on the shard
 }
 
+pub enum GarbageCollectorStatus {
+    GarbageCollected,
+    TryLater,
+}
+
```

Comment:
if you implement:
```rust
impl From<GarbageCollectorStatus> for nucliadb_core::protos::garbage_collector_response::Status {
   .....
}
```
The implementation of `gc` gets less noisy.

Reply:
Done